install/resources/upgrade49.php

```diff
@@ -35,6 +35,9 @@ function upgrade49_dbchanges()
 
 	$db->delete_query("settings", "name='allowyahoofield'");
 
+	$db->update_query('profilefields', array(
+		'description' => "REPLACE(description, 'sex', 'gender')",
+	), 'fid = 3');
```

Comment:
This database upgrade script `upgrade49.php` is for 1.8.21 to 1.8.22. If this PR is going to be merged before 1.8.23 release, changes should be made accordingly in `upgrade50.php`.

Reply:
See the above comment by @Sama34 - we want to remove this upgrade script completely.

Reply:
> See the above comment by @Sama34 - we want to remove this upgrade script completely.

Makes sense to only affect new installs. I vote for this. Maybe existing forums have already altered those fields.

Reply:
I think this has been addressed? Hoping you can double check.